src/plugins/discover/public/application/angular/discover.js

```diff
@@ -814,7 +816,7 @@ function discoverController($element, $route, $scope, $timeout, $window, Promise
         if (error instanceof Error && error.name === 'AbortError') return;
 
         $scope.fetchStatus = fetchStatuses.NO_RESULTS;
-        $scope.rows = [];
+        $scope.fetchError = error;
 
         data.search.showError(error);
```

Comment:
There's an annoying problem here, that if the error is a `SearchTimeoutError` - it's won't show again.
I'm not sure it's worth fixing, but at least it might be worth documenting.

Reply:
It's just displayed once in a session?

Reply:
@lizozom but the error is displayed as popover, right?

Reply:
Yeah, but that popover will be displayed only once. Since you're re-calling shower to, it doesn't get shown. Not sure if I have a good proposition for fixing it, but I don't think it's critical.

Reply:
I see, so the replay doesn't work, also think it's not critical, the button is an additional option if you have missed the toast. would be interesting why it's just callable once.